src/scenes/DeliverablesRouter/SpeciesDeliverableView.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useMemo } from 'react';
+
+import { Box, Typography, useTheme } from '@mui/material';
+import { BusySpinner } from '@terraware/web-components';
+
+import { Crumb } from 'src/components/BreadCrumbs';
+import Metadata from 'src/components/DeliverableView/Metadata';
+import MobileMessage from 'src/components/DeliverableView/MobileMessage';
+import TitleBar from 'src/components/DeliverableView/TitleBar';
+import { EditProps, ViewProps } from 'src/components/DeliverableView/types';
+import Page from 'src/components/Page';
+import Card from 'src/components/common/Card';
+import Button from 'src/components/common/button/Button';
+import { APP_PATHS } from 'src/constants';
+import { useLocalization } from 'src/providers';
+import RejectedDeliverableMessage from 'src/scenes/DeliverablesRouter/RejectedDeliverableMessage';
+import strings from 'src/strings';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+
+export type Props = EditProps & {
+  isBusy?: boolean;
+};
+
+const SpeciesDeliverableView = (props: Props): JSX.Element => {
+  const { ...viewProps }: ViewProps = props;
+  const { isMobile } = useDeviceInfo();
+  const { activeLocale } = useLocalization();
+  const theme = useTheme();
+
+  // TODO: get participant project species for participant project id
+  // and disable button if no species OR if all species are already approved
```

Comment:
Planning to do this once [SW-5272](https://github.com/terraware/terraware-web/pull/2555) is merged.